react/Coupon.tsx

```diff
@@ -82,21 +86,21 @@ const Coupon: StorefrontFunctionComponent<CouponProps> = ({
     <Fragment>
       {showPromoButton ? (
         <Fragment>
-          {!currentCoupon && (
+          {!coupon && (
             <div>
```

Comment:
Useful `div`?

Reply:
Unfortunately, it is. 

Without the `div`, the `ButtonPlain` component doesn't behave properly. It takes the whole summary width. See below

![image](https://user-images.githubusercontent.com/12574426/68802987-ca98ba80-063d-11ea-9844-3f502a8418f8.png)